src/Symfony/Component/Translation/Exception/UnsupportedSchemeException.php

```diff
@@ -21,6 +21,10 @@ class UnsupportedSchemeException extends LogicException
             'class' => Bridge\Loco\Provider\LocoProviderFactory::class,
             'package' => 'symfony/loco-translation',
         ],
+        'crowdin' => [
+            'class' => Bridge\Crowdin\Provider\CrowdinProviderFactory::class,
+            'package' => 'symfony/crowdin-translation',
```

Comment:
```suggestion
            'package' => 'symfony/crowdin-translation-provider',
```
but depends on decision in https://github.com/symfony/symfony/pull/40944/files#r620020546